e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,212 @@
+	kubectl_bin patch "service/$svc_name" --type=json --patch '[
+        {
+            "op": "add",
+            "path": "/metadata/labels",
+            "value": {
+                "notIgnoredLabel": "true",
+                "ignoredLabel": "true"
+            }
+        },
+        {
+            "op": "add",
+            "path": "/metadata/annotations",
+            "value": {
+                "notIgnoredAnnotation": "true",
+                "ignoredAnnotation": "true"
+            }
+        }]'
+
+	sleep 10 # waiting for reconcile. 5 seconds it is not enough in some cases.
+	desc "check if annotations/labels are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-manual"
+
+	desc "adding ignoreAnnotations and ignoreLabels"
+	kubectl_bin patch pxc "$cluster" --type=json --patch '[
+        {
+            "op": "add",
+            "path": "/spec/ignoreAnnotations",
+            "value": ["ignoredAnnotation"]
+        },
+        {
+            "op": "add",
+            "path": "/spec/ignoreLabels",
+            "value": ["ignoredLabel"]
+        }
+    ]'
+	sleep 10
+	desc "check if annotations/labels are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-manual"
+
+	desc "adding serviceLabels and serviceAnnotations to $expose_path"
+	kubectl_bin patch pxc ${cluster} --type=json --patch '[
+        {
+            "op": "replace",
+            "path": "'$expose_path'/'$label_path'",
+			"value": {
+				"crLabel": "true",
+			}
+        },
+        {
+            "op": "replace",
+            "path": "'$expose_path'/'$annotation_path'",
+			"value": {
+				"crAnnotation": "true",
+			}
+        }]'
+	sleep 10
+	desc "check if not ignored labels and annotations are deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-crmeta"
+
+	desc "removing serviceLabels and serviceAnnotations from $expose_path"
+	kubectl_bin patch pxc ${cluster} --type=json --patch '[
+        {
+            "op": "remove",
+            "path": "'$expose_path'/'$annotation_path'"
+        }]'
+  if [ ${service_type} != "pxc" ]; then
+    kubectl_bin patch pxc ${cluster} --type=json --patch '[
+          {
+              "op": "remove",
+              "path": "'$expose_path'/'$label_path'"
+          }]'
+  fi
+	sleep 10
+	desc "check if labels and annotations are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-crmeta"
+
+	desc "adding other serviceLabels and serviceAnnotations to $expose_path"
+	kubectl_bin patch pxc ${cluster} --type=json --patch '[
+        {
+            "op": "replace",
+            "path": "'$expose_path'/'$label_path'",
+			"value": {
+				"otherCrLabel": "true",
+				"secondCrLabel": "true",
+			}
+        },
+		{
+            "op": "replace",
+            "path": "'$expose_path'/'$annotation_path'",
+			"value": {
+				"otherCrAnnotation": "true",
+				"secondCrAnnotation": "true",
+			}
+		}]'
+	sleep 25
```

Comment:
I know that you asked) 
Because for some reason for proxysql we not always can update service inside less sleep. So I increased it to make test more stable